domdiv/__init__.py

```diff
@@ -175,8 +205,10 @@ def parse_opts(arglist):
         default=-1,
         help="stop generating after this many pages, -1 for all")
     parser.add_argument("--language",
-                        default='en_us',
-                        help="language of card texts")
+                        default=LANGUAGE_DEFAULT,
```

Comment:
can we add an actual `choices=LANGUAGE_CHOICES` or similar? Right now the script dies with an exception if you specify a non-existent language, and it'd be nice if it provided a better help message.